src/libexpr/eval.cc

```diff
@@ -144,8 +154,10 @@ string showType(const Value & v)
         case tApp: return "a function application";
         case tLambda: return "a function";
         case tBlackhole: return "a black hole";
-        case tPrimOp: return "a built-in function";
-        case tPrimOpApp: return "a partially applied built-in function";
+        case tPrimOp:
+            return "the built-in function " + string(v.primOp->name);
+        case tPrimOpApp:
+            return "the partially applied built-in function " + string(getPrimOp(v)->primOp->name);
```

Comment:
Can you add single quotes around the function name, e.g.
```
return fmt("the built-in function '%s'", string(v.primOp->name));
```

Reply:
Done, thanks for the feedback!